scripts/prepublish.sh

```diff
@@ -30,8 +29,11 @@ rm abis.json
 
 rm deployments.json 
 
-cat << EOF > index.ts
-import abis from "./publish/abis";
-import addresses from "./publish/addresses";
+cat << EOF > publish/index.ts
+import abis from "./abis";
+import addresses from "./addresses";
 export { abis, addresses };
 EOF
+
+# Run TypeScript compilation
+tsc ./publish/index.ts
```

Comment:
To be safe (for if this script is run on systems without `tsc` already installed), should this line be something like `npx tsc ./publish/index.ts`?

Reply:
I can add it to be safe